scripts/badger/process_bribes_graviaura.py

```diff
@@ -55,29 +56,60 @@ def claim_and_sell_for_weth():
 
 def sell_weth():
     weth_total = WETH.balanceOf(PROCESSOR)
-    badger_share = int(WETH.balanceOf(PROCESSOR) * BADGER_SHARE)
-    aura_share = int(WETH.balanceOf(PROCESSOR) - badger_share)
+    badger_total = BADGER.balanceOf(PROCESSOR)
```

Comment:
while the calculation for the splits lgtm, i am thinking that `badger_total` will be always equal to zero in the processor balance

due to this [logic](https://etherscan.io/address/0x0d724e8aee6f73b35a596c8c947c92c75eac7818#code#F1#L2075), the badger tokens claimed will never reach the processor, will be emitted into the tree and will not allow to carry this calculations, unless i am missing something

perhaps this calculation could be carry over, but needs to leverage the information of the [endpoint](https://hhand.xyz/reward/1/0x3c0989eF27e3e3fAb87a2d7C38B35880C90E63b5) instead

Reply:
Good catch, this would be the case indeed.

Fetching the BADGER bribes amount from the endpoint should be the way to go and I think we should still do so to ensure that the total BADGER emitted meets the 25/75 split. Will apply this change.

Reply:
> Good catch, this would be the case indeed.
> 
> Fetching the BADGER bribes amount from the endpoint should be the way to go and I think we should still do so to ensure that the total BADGER emitted meets the 25/75 split. Will apply this change.

NOTE: We need to see what happens on a second bribe round, as I believe `amount` will be an increasing amount (not the amount for that round but the total) due to how HH code is written

Reply:
Question:

> while the calculation for the splits lgtm, i am thinking that `badger_total` will be always equal to zero in the processor balance
> 
> due to this [logic](https://etherscan.io/address/0x0d724e8aee6f73b35a596c8c947c92c75eac7818#code#F1#L2075), the badger tokens claimed will never reach the processor, will be emitted into the tree and will not allow to carry this calculations, unless i am missing something
> 
> perhaps this calculation could be carry over, but needs to leverage the information of the [endpoint](https://hhand.xyz/reward/1/0x3c0989eF27e3e3fAb87a2d7C38B35880C90E63b5) instead

Wouldn't the `badger_total` be non-zero if BADGER is a bribe?

Reply:
> NOTE: We need to see what happens on a second bribe round, as I believe amount will be an increasing amount (not the amount for that round but the total) due to how HH code is written

@GalloDaSballo, interesting, we should definitely confirm this as we also rely on this quantity to claim bribes and swap them for WETH.

Reply:
> Wouldn't the badger_total be non-zero if BADGER is a bribe?

As Petro pointed out, graviAURA is [hardcoded](https://etherscan.io/address/0x0d724e8aee6f73b35a596c8c947c92c75eac7818#code#F1#L2075) to directly transfer BADGER bribes to the Tree so they wouldn't fall into the processor at any point.

Reply:
> > NOTE: We need to see what happens on a second bribe round, as I believe amount will be an increasing amount (not the amount for that round but the total) due to how HH code is written
> 
> @GalloDaSballo, interesting, we should definitely confirm this as we also rely on this quantity to claim bribes and swap them for WETH.

i will say that if we claim with the right cadence, post-round, it should not become an issue that amounts gets accumulated under the same bribed token cc: @GalloDaSballo

Reply:
> Good catch, this would be the case indeed.
> 
> Fetching the BADGER bribes amount from the endpoint should be the way to go and I think we should still do so to ensure that the total BADGER emitted meets the 25/75 split. Will apply this change.

also i am thinking that the endpoint, likely will clear it out after claiming in step 1, so it may be worthy to save the claimable amount somewhere ( json, introduce as a param in the second step etc), otherwise, i think later on when trying to ping the endpoint, will come back empty data, since there is nothing to claim anymore

Reply:
> also i am thinking that the endpoint, likely will clear it out after claiming in step 1, so it may be worthy to save the claimable amount somewhere ( json, introduce as a param in the second step etc), otherwise, i think later on when trying to ping the endpoint, will come back empty data, since there is nothing to claim anymore

This is what I thought would happen, hard to know until we try but it makes sense. I like the idea of storing on a JSON, will incorporate this.